aten/src/ATen/native/TensorIterator.h

```diff
@@ -231,8 +231,8 @@ struct CAFFE2_API TensorIterator {
   void narrow(int dim, int64_t start, int64_t size);
   /// Narrows every dim after and including `start_dim` to size one.
   void select_all_keeping_dim(int start_dim, IntArrayRef starts);
-  /// Replaces the data pointer and strides for the operand at index `arg`
-  void replace_operand(int arg, void* data, IntArrayRef stride);
+  /// Replaces the data pointer for the operand at index `arg`
+  void replace_operand(int arg, void* data);
```

Comment:
This function wasn't used before. Changing the `stride` seems wrong, as the stride needs to be in sync with the dimensions, and the original values could have been reordered/coalesced in build.